src/android/app/src/main/res/menu/menu_emulation.xml

```diff
@@ -3,125 +3,125 @@
+<!--        android:id="@+id/menu_emulation_switch_screen_layout"-->
+<!--        app:showAsAction="never"-->
+<!--        android:title="@string/emulation_switch_screen_layout">-->
+<!--&lt;!&ndash;        <menu>&ndash;&gt;-->
+<!--&lt;!&ndash;            <group android:checkableBehavior="single">&ndash;&gt;-->
+<!--&lt;!&ndash;                <item&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:id="@+id/menu_portrait_layout_top_full"&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:title="@string/emulation_screen_layout_landscape" />&ndash;&gt;-->
+
+<!--&lt;!&ndash;                <item&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:id="@+id/menu_screen_layout_portrait"&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:title="@string/emulation_screen_layout_portrait" />&ndash;&gt;-->
+
+<!--&lt;!&ndash;                <item&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:id="@+id/menu_screen_layout_single"&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:title="@string/emulation_screen_layout_single" />&ndash;&gt;-->
+
+<!--&lt;!&ndash;                <item&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:id="@+id/menu_screen_layout_sidebyside"&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:title="@string/emulation_screen_layout_sidebyside" />&ndash;&gt;-->
+
+<!--&lt;!&ndash;                <item&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:id="@+id/menu_screen_layout_hybrid"&ndash;&gt;-->
+<!--&lt;!&ndash;                    android:title="@string/emulation_screen_layout_hybrid" />&ndash;&gt;-->
+<!--&lt;!&ndash;            </group>&ndash;&gt;-->
+<!--&lt;!&ndash;        </menu>&ndash;&gt;-->
+<!--    </item>-->
+
+<!--    <item-->
+<!--        android:id="@+id/menu_emulation_swap_screens"-->
+<!--        app:showAsAction="never"-->
+<!--        android:title="@string/emulation_swap_screens"-->
+<!--        android:checkable="true" />-->
+
+<!--    <item-->
+<!--        android:id="@+id/menu_emulation_show_fps"-->
+<!--        app:showAsAction="never"-->
+<!--        android:title="@string/emulation_show_fps"-->
+<!--        android:checkable="true" />-->
+
+<!--    <item-->
+<!--        android:id="@+id/menu_emulation_show_overlay"-->
+<!--        app:showAsAction="never"-->
+<!--        android:title="@string/emulation_show_overlay"-->
+<!--        android:checkable="true" />-->
+
+<!--    <item-->
+<!--        android:id="@+id/menu_emulation_open_cheats"-->
+<!--        app:showAsAction="never"-->
+<!--        android:title="@string/emulation_open_cheats" />-->
+
+<!--    <item-->
+<!--        android:id="@+id/menu_emulation_open_settings"-->
+<!--        app:showAsAction="never"-->
+<!--        android:title="@string/emulation_open_settings" />-->
+
+<!--    <item-->
+<!--        android:id="@+id/menu_emulation_close_game"-->
+<!--        app:showAsAction="never"-->
+<!--        android:title="@string/emulation_close_game" />-->
```

Comment:
What's the deal with this file? Why is everything commented out?

Reply:
oh i forgot about this. I commented this out to check if this file was actually being used anywhere when trying to better understand the menu structures. Turns out it isn't! Somebody must have used this xml file to create a mockup of the emulation menu at some point, but the actual menu is generated dynamically entirely in code, this file is useless. I'll commit a delete.

TBH, I think that we probably *should* be using an xml file for this menu, rather than generating everything in code, but my Android UI knowledge is "just enough to tweak, not enough to design from scratch" levels at this point, so there may be a logical reason the original code went in this direction.

Reply:
I'd just remove it now